tigerblood.go

```diff
@@ -140,6 +140,14 @@ func (h *TigerbloodHandler) CreateReputation(w http.ResponseWriter, r *http.Requ
 		log.Printf("Could not unmarshal request body: %s", err)
 		return
 	}
+
+	ip := net.ParseIP(entry.IP)
+	if ip == nil {
```

Comment:
Since you're not using the value, maybe just `if net.ParseIP(entry.IP) == nil {`